.rubocop.yml

```diff
@@ -1,14 +1,20 @@
 AllCops:
   Exclude:
-    - "Guardfile"
-    - "Rakefile"
+    - 'Guardfile'
+    - 'Rakefile'
 
   DisplayCopNames: true
 
 Layout/LineLength:
   Max: 120
+Metrics/ModuleLength:
+  Enabled: false
+Metrics/CyclomaticComplexity:
+  Enabled: false
+Metrics/PerceivedComplexity:
+  Enabled: false
 Metrics/MethodLength:
-  Max: 20
+  Max: 30
```

Comment:
- [ ] You were asked not to modify the linter rules. It is not good practice to do so.  Instead, for this project you should disable the cops where necessary. Please take a look at this [article](https://rubydoc.info/gems/rubocop/0.24.1/frames#Disabling_Cops_within_Source_Code) for more information.

Reply:
yeah, the thing is that on my first review i got asked to bodify de yml file instead onf disableing the rules

Reply:
Sorry you were asked to do that! Your reviewer should not have asked you to do that. As a rule of thumb, you shouldn’t modify the linter rules. Instead you should try to solve the issue or in some cases you can disable the rules. Great job overall and good luck on your next project! 🍀